Disher/View Controllers/RecipeInListViewController.m

```diff
@@ -24,7 +24,9 @@ - (void)viewDidLoad {
     if ([self.passedRecipe.source isEqualToString:@"Spoonacular"]) {
         [Recipe getRecipeInfo:self.passedRecipe.recipeID withSource:@"Spoonacular" withCompletion:^(NSDictionary * _Nonnull recipeInformation) {
             self.dishTitleLabel.text = [recipeInformation objectForKey:@"title"];
-            self.descriptionLabel.text = [self flattenHtml:[recipeInformation objectForKey:@"instructions"]];
+            if ([recipeInformation objectForKey:@"instructions"] != [NSNull null]) {
```

Comment:
should this be
```
if ([recipeInformation objectForKey:@"instructions"] != nil) {
```
usually you would use `[NSNull null]` to check for nullability on C pointers. on Objective-C you would typically use `nil`, as the nullability checker for "the pointer pointing to null". in this case it seems that `recipeInformation` is a dictionary, an an Objective-C dictionary's key and value would be an Objective-C object.

check https://nshipster.com/nil/

Reply:
My app crashes when I encounter a null value for the instructions in Spoonacular so I used [NSNull null].